{"name": "test_5[7-8-9]", "status": "failed", "statusDetails": {"message": "AssertionError: assert ((7 * 8) + 9) == 5", "trace": "x = 7, y = 8, z = 9\n\n    @pytest.mark.parametrize(\"x, y, z\", [(1,2,3), (4,5,6),(7,8,9),(1,2,3)])\n    def test_5(x, y, z):\n>       assert x * y + z == 5\nE       assert ((7 * 8) + 9) == 5\n\ntest_4.py:6: AssertionError"}, "parameters": [{"name": "x", "value": "7"}, {"name": "y", "value": "8"}, {"name": "z", "value": "9"}], "start": 1639816810268, "stop": 1639816810268, "uuid": "88859a04-b03e-4e5a-8b1d-319ef3c58b4a", "historyId": "b12f2592ae45e7b025034d548be81cd4", "testCaseId": "54c5933bbf5498919612b4ec9a952f13", "fullName": "test_4#test_5", "labels": [{"name": "suite", "value": "test_4"}, {"name": "host", "value": "Ani-PC"}, {"name": "thread", "value": "8160-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_4"}]}